src/lib/converter/plugins/SourcePlugin.ts

```diff
@@ -104,11 +104,14 @@ export class SourcePlugin extends ConverterComponent {
         if (!reflection.sources) {
             reflection.sources = [];
         }
+
+        const sources = reflection.sources;
+
         if (reflection instanceof DeclarationReflection) {
             file.reflections.push(reflection);
         }
 
-        reflection.sources.push({
+        sources.push({
```

Comment:
This change is needed, because `file.reflections.push(reflection)` can potentially mutate `sources` key. 

I guess, typescript 3.4 became smarter and requires us to have an explicit stored reference to the array.